src/pages/trade-types/margin/_start-trading.js

```diff
@@ -32,7 +32,15 @@ const StartTrading = () => {
                             )}
                         </Timeline.Item>
                     </Timeline>
-                    <LinkButton mt="4rem" to="/dmt5/" secondary="true">
+                    <LinkButton
+                        mt="4rem"
+                        external="true"
+                        type="mt5"
+                        to="#demo"
```

Comment:
```suggestion
                        to="/#demo"
```

Reply:
You will need to include a slash here as the urls from `type` does not end with a trailing slash.

Reply:
The format that I need is app.deriv.com/mt5#demo as per card, thus no slash will be added for `type` for now @sean-binary @yashim-deriv

Reply:
Oh that's right 👍  my bad.